<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1411</width>
     <height>771</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_main">
    <item>
     <layout class="QHBoxLayout" name="layout_item">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="lbl_dept">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>작업부서</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comb_dept_name">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_dept_id">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>조 회</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tbl_info">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
